scheduler/core/plans/__init__.py

```diff
@@ -69,6 +68,10 @@ def __init__(self, night_events: Mapping[Site, NightEvents], night_idx: int):
                                         site,
                                         len(ne.times[night_idx]))
 
+    @property
+    def time_slot_length(self):
+        return 0
```

Comment:
Why return 0 here?

Reply:
Ooops. I have no idea why I did this. *doh* I'll fix this.